src/app/pages/auth/login/page.tsx

```diff
@@ -0,0 +1,139 @@
+"use client";
```

Comment:
File naming pattern is wrong. Get rid of pages directory as it is useless. The pattern should be `src/app/auth/login/page.tsx`